controller/user/manager.py

```diff
@@ -20,6 +20,7 @@ def get_or_create_user(user_id: str) -> User:
     if not user_item:
         user_item = user.create(user_id, with_commit=True)
         kratos.__refresh_identity_cache()
+        migrate_kratos_users()
```

Comment:
the update of our db is only called if a user doesn't exist yet. This means on startup & after a new user we get the update potentially resulting in an issue for updates on existing users (e.g. verfied status) I think adding it ot eh __refresh_identity_cache function itself will solve this since it's called on a timer

Reply:
The migrate_kratos_users() called the get_cached_users() which calls __refresh_identity_cache and if we add in in it it comes to a loop, that's why i added it here

Reply:
then we need to change the structure a bit imo.